src/benzina/torch/operations.py

```diff
@@ -377,11 +375,27 @@ def __call__(self, index, in_shape, out_shape, rng):
         """Return a random similarity transformation."""
         s = np.exp(rng.uniform(low=np.log(self.s[0]), high=np.log(self.s[1])))
         if self.ar is not None:
-            ar = np.exp(rng.uniform(low=np.log(self.ar[0]),
-                                    high=np.log(self.ar[1])))
-            crop_area = s * in_shape[0] * in_shape[1]
-            crop_w = np.sqrt(crop_area * ar)
-            crop_h = np.sqrt(crop_area / ar)
+            for _ in range(10):
```

Comment:
I'm not sure if we need to try 10 times before fallback to a center crop, but this is what torchvision does in [RandomResizedCrop](https://pytorch.org/docs/stable/_modules/torchvision/transforms/transforms.html#RandomResizedCrop) so maybe it's better to start with this